src/modules/users/components/AvatarDropdown/AvatarDropdown.tsx

```diff
@@ -66,13 +67,19 @@ const AvatarDropdown = ({ onlyLogout = false }: Props) => {
         ],
       }}
     >
-      <button
-        className={styles.avatarButton}
-        type="button"
-        data-test="avatarDropdown"
-      >
-        <UserAvatar address={walletAddress} notSet={ethereal} size="s" />
-      </button>
+      {({ isOpen, toggle, ...props }) => (
+        <button
+          className={classnames(styles.avatarButton, {
+            [styles.activeDropdown]: isOpen,
+          })}
+          onClick={toggle}
+          type="button"
+          data-test="avatarDropdown"
+          {...props}
+        >
+          <UserAvatar address={walletAddress} notSet={ethereal} size="s" />
+        </button>
+      )}
```

Comment:
You introduce some react standardization errors in here:

![Screenshot from 2021-02-05 14-34-56](https://user-images.githubusercontent.com/1193222/107036833-e5b39500-67c2-11eb-96e6-2167afe9ebc2.png)

Please investigate and fix it.